.prettierrc

```diff
@@ -0,0 +1,4 @@
+{
+    "printWidth": 70,
```

Comment:
I would consider 80 columns, but that's not a huge deal. 

Python's [PEP 8 used 80](https://www.python.org/dev/peps/pep-0008/#maximum-line-length).

Reply:
JSX might make people want to push to 100. 

Totally open to discussing this.

Reply:
Answer below ... ⬇️

Reply:
I'll defer to your answer LOL
>source files for cyverse-ui will be included in the Style Guide (./guide within this repository). If you consider the guidelines for PEP8, comments have recommended column width of 72 for what I believe is a reason in that you'll be displaying it with a navigation bar to the one side. For PEP8, I think the reasoning is that you'll be commenting for a Docstring (__doc__) for a class, that puts you two levels indented (8 spaces, so 80-8 = 72).

Honestly, deep nesting is something I avoid and also will break up components into small pieces so 70 has been working well. I want to force inlining on long expressions as much as I can.

Reply:
72 works too